doc/qbk/05_performance.qbk

```diff
@@ -0,0 +1,282 @@
+[/
+    Copyright (c) 2019 Vinnie Falco (vinnie.falco@gmail.com)
+
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+    Official repository: https://github.com/cppalliance/json
+]
+
+[section Performance]
```

Comment:
when you rename sections you break all the hyperlinks.
This is the current link:
```
http://vinniefalco.github.io/doc/json/json/benchmarks.html
```
And this will be the link with your changes:
```
http://vinniefalco.github.io/doc/json/json/performance.html
```
So... all those reddit posts, Twitter posts, github issues in foreign repos, Boost mailing list posts, etc... that link to the benchmarks.html page will have broken links.

Reply:
Yes, that is why I did this before release. I don't think that there are too many links out there anyways, but we can always add a redirect anyways.

Reply:
What is wrong with "Benchmarks?"

Reply:
It doesn't leave much room for expanding the section in the future.

Reply:
Expanding how, to talk about performance? Some kind of exposition? How we achieved our numbers?

Reply:
Yes

Reply:
If you're going to write exposition it should go in another section. So you can leave this one at "Benchmarks", and later add a "Performance" section where you have a larger discussion.

Reply:
Yes I see, when the time comes we can change "Benchmarks" in the top level index to "Performance", put the exposition there, and the current "Benchmarks" page will be a sub-section on its own page (and thus preserve the Link). When we are ready to add the exposition I will show you how to structure the documents.